lib/internal/errors.js

```diff
@@ -178,6 +178,16 @@ E('ERR_TRANSFORM_WITH_LENGTH_0',
 E('ERR_UNKNOWN_SIGNAL', 'Unknown signal: %s');
 E('ERR_UNKNOWN_STDIN_TYPE', 'Unknown stdin file type');
 E('ERR_UNKNOWN_STREAM_TYPE', 'Unknown stream file type');
+E('ERR_SOCKET_CLOSED', 'Socket is closed');
+E('ERR_TLS_DH_PARAM_SIZE', (size) =>
+  `DH parameter size ${size} is less than 2048`);
+E('ERR_TLS_HANDSHAKE_TIMEOUT', 'TLS handshake timeout');
+E('ERR_TLS_SESSION_ATTACK', 'TSL session renegotiation attack detected');
+E('ERR_MULTIPLE_CALLBACK',
```

Comment:
The change to `test-stream-transform-callback-twice.js` is good by me, but IMHO you should just remote L162, since this line simply overrides it anyway (there's no other magic).